files/etc/xrd/get_non_isolated_cores.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
```

Comment:
Does this have to be bash? I imagine it would be a lot simpler if it were in python